drydock_backups/plugin.py

```diff
@@ -0,0 +1,237 @@
+from __future__ import annotations
+
+import os
+import os.path
+from glob import glob
+
+import click
+import pkg_resources
+from tutor import hooks
+
+from .__about__ import __version__
+
+config = {
+    "defaults": {
+        "VERSION": __version__,
+        "IMAGE": "ednxops/shipyard-utils",
+        "CRON_SCHEDULE": '0 2 * * *',
+        "AWS_ACCESS_KEY": "",
+        "AWS_SECRET_KEY": "",
+        "BUCKET_NAME": "",
+        "BUCKET_PATH": "backups",
+        "ENDPOINT": None,
```

Comment:
`ENDPOINT` is a bit confusing, what about `CUSTOM_STORAGE_ENDPOINT`? (the final name of the variable would be `BACKUP_CUSTOM_STORAGE_ENDPOINT`)